ethToElrond/v2/ethToElrond/steps/stepsFactory_test.go

```diff
@@ -0,0 +1,29 @@
+package steps
+
+import (
+	"testing"
+
+	"github.com/ElrondNetwork/elrond-eth-bridge/ethToElrond/v2/ethToElrond"
+	"github.com/ElrondNetwork/elrond-eth-bridge/testsCommon/bridgeV2"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func TestCreateSteps_Errors(t *testing.T) {
+	t.Parallel()
+
+	steps, err := CreateSteps(nil)
+
+	assert.Nil(t, steps)
+	assert.Equal(t, ethToElrond.ErrNilExecutor, err)
+}
+
+func TestCreateSteps_ShouldWork(t *testing.T) {
+	t.Parallel()
+
+	steps, err := CreateSteps(bridgeV2.NewEthToElrondBridgeStub())
+
+	require.NotNil(t, steps)
+	require.Nil(t, err)
+	require.Equal(t, 6, len(steps))
```

Comment:
check the exact order as well?

Reply:
you cannot check the order here since it is a map , the order will be checked in the following semi-integrated tests.